keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -693,6 +693,9 @@ in back-to-back releases.
 
 #### Beta
 - e2e tests pass with containerd, CRI-O, and cri-dockerd
+ - https://github.com/containerd/containerd/pull/9787
+ - https://github.com/cri-o/cri-o/pull/7962
+ - https://github.com/Mirantis/cri-dockerd/pull/370
```

Comment:
We should also add e2e tests to main kuberentes repo pulling in updated container runtime[s].

Reply:
We already have: https://github.com/kubernetes/kubernetes/blob/v1.30.1/test/e2e_node/mount_rro_linux_test.go